### DISCUSSION TITLE
Group Policy to control default Terminal setting

### DISCUSSION THREAD

**Discussion by htcfreek** [microsoft/terminal#18303] [Q&A] [answered]
Since a year or longer I can choose if WT is the default conhost.

Is it possible to implement a policy that allows company admins to control this centralized?

**Comment by DHowett** [comment, marked as answer]
Right now, enterprise administrators can use Group Policy to roll out registry changes as documented in [this knowledgebase article](https://support.microsoft.com/en-us/windows/command-prompt-and-windows-powershell-for-windows-11-6453ce98-da91-476f-8651-5c14d5777c20).

**Reply by htcfreek** [reply]
Two questions:
1.Why not putting this into the group policy admx template using a drop down policy? (That would make configuration much easier.)
2. The docs tell me to create the values under `HKCU\Console\%%Startup`. Is this really correct? The two % signs looking strange. 🤔
